src/types/onyx/ReportNextStep.ts

```diff
@@ -0,0 +1,50 @@
+import {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+
+type Message = {
+    text: string;
+    type: string;
+    action: string;
+};
+
+type Button = {
+    text?: string;
+    tooltip?: string;
+    disabled?: boolean;
+    hidden?: boolean;
+    data?: unknown[];
+};
+
+type ReportNextStep = {
+    /** The message parts of the next step */
+    message: Message[];
+
+    /** The title for the next step */
+    title?: string;
+
+    /** Whether the user should take some sort of action in order to unblock the report */
+    requiresUserAction?: boolean;
+
+    /** The type of next step */
+    type: ValueOf<typeof CONST.NEXT_STEP.type>;
```

Comment:
```suggestion
    type?: 'alert' | 'neutral' | null;
```

I think you can just specify the possible options here instead of defining them on CONST. Also, we should make the prop optional because isn't required in the propTypes file.